From 51f8931e0e47ac8968ef1a84c86b483b44895d16 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Sat, 14 Apr 2018 22:24:20 +0200
Subject: [PATCH 23/23] Work around path issue with generated inspector
 protocol files

Apparently the SHARED_INTERMEDIATE_DIR is expected to be an absolute
path, but is not...

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 deps/v8/src/inspector/inspector.gypi | 30 ++++++++++++++--------------
 1 file changed, 15 insertions(+), 15 deletions(-)

diff --git a/deps/v8/src/inspector/inspector.gypi b/deps/v8/src/inspector/inspector.gypi
index bb96b0f6..b31c44fc 100644
--- a/deps/v8/src/inspector/inspector.gypi
+++ b/deps/v8/src/inspector/inspector.gypi
@@ -5,21 +5,21 @@
 {
   'variables': {
     'inspector_generated_sources': [
-      '<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/Forward.h',
-      '<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/Protocol.cpp',
-      '<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/Protocol.h',
-      '<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/Console.cpp',
-      '<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/Console.h',
-      '<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/Debugger.cpp',
-      '<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/Debugger.h',
-      '<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/HeapProfiler.cpp',
-      '<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/HeapProfiler.h',
-      '<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/Profiler.cpp',
-      '<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/Profiler.h',
-      '<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/Runtime.cpp',
-      '<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/Runtime.h',
-      '<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/Schema.cpp',
-      '<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/Schema.h',
+      '../../../../<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/Forward.h',
+      '../../../../<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/Protocol.cpp',
+      '../../../../<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/Protocol.h',
+      '../../../../<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/Console.cpp',
+      '../../../../<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/Console.h',
+      '../../../../<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/Debugger.cpp',
+      '../../../../<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/Debugger.h',
+      '../../../../<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/HeapProfiler.cpp',
+      '../../../../<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/HeapProfiler.h',
+      '../../../../<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/Profiler.cpp',
+      '../../../../<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/Profiler.h',
+      '../../../../<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/Runtime.cpp',
+      '../../../../<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/Runtime.h',
+      '../../../../<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/Schema.cpp',
+      '../../../../<(SHARED_INTERMEDIATE_DIR)/src/inspector/protocol/Schema.h',
       '<(SHARED_INTERMEDIATE_DIR)/include/inspector/Debugger.h',
       '<(SHARED_INTERMEDIATE_DIR)/include/inspector/Runtime.h',
       '<(SHARED_INTERMEDIATE_DIR)/include/inspector/Schema.h',
-- 
2.17.0.windows.1

